dask_geopandas/io/parquet.py

```diff
@@ -0,0 +1,126 @@
+from functools import partial
+from typing import TYPE_CHECKING
+
+import pandas as pd
+
+import geopandas
+
+import dask.dataframe as dd
+
+try:
+    # pyarrow is imported here, but is an optional dependency
+    from dask.dataframe.io.parquet.arrow import ArrowEngine
+except ImportError:
+    ArrowEngine = object
+
+if TYPE_CHECKING:
+    import pyarrow
+
+
+class GeoArrowEngine(ArrowEngine):
+    @classmethod
+    def read_metadata(cls, *args, **kwargs):
+        meta, stats, parts, index = super().read_metadata(*args, **kwargs)
+
+        # Update meta to be a GeoDataFrame
+        # TODO convert columns based on GEO metadata (this will now only work
+        # for a default "geometry" column)
+        meta = geopandas.GeoDataFrame(meta)
+
+        return (meta, stats, parts, index)
+
+    @classmethod
+    def _arrow_table_to_pandas(
+        cls, arrow_table: "pyarrow.Table", categories, **kwargs
+    ) -> pd.DataFrame:
+        from geopandas.io.arrow import _arrow_to_geopandas
+
+        _kwargs = kwargs.get("arrow_to_pandas", {})
+        _kwargs.update({"use_threads": False, "ignore_metadata": False})
+
+        # TODO support additional keywords
+        return _arrow_to_geopandas(arrow_table)
+
+    @classmethod
+    def _pandas_to_arrow_table(
+        cls, df: pd.DataFrame, preserve_index=False, schema=None
+    ) -> "pyarrow.Table":
+        from geopandas.io.arrow import _geopandas_to_arrow
```

Comment:
Why is the import in the method? Can we move it out and wrap it in a try except depending on pyarrow availablitity

Reply:
That's also a possibility, but then I need to add `_geopandas_to_arrow = None` in the except clause (and the same for the other method), so I personally find this cleaner, but if you prefer that, can certainly change it

Reply:
No that makes sense. I am happy to leave it.